release_notes/topics/product/7_4_final.adoc

```diff
@@ -47,7 +50,9 @@ Administrators can see all credentials of a particular user and some public meta
 
 Users can see all their credentials in the Account Console and add or delete credentials. See the section on the New Account Console, which is a Technology Preview feature. The currently supported account console, the User Account Service, does not support this feature. It supports OTP in a similar way to the previous RH-SSO version.
 
-See link:{adminguide_authentication_flow_link}[{adminguide_authentication_flow_name}] in the link:{adminguide_link}[{adminguide_name}] for more details.
+[role="_additional-resources"]
+.Additional resources
+* link:{adminguide_authentication_flow_link}[Server Administration Guide]
```

Comment:
The link it is redirecting to is https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/server_administration_guide/index#_authentication-flows which is not redirecting to the corresponding topic. Please change it to /index#authentication-flows at the end

Reply:
Okay, I redefined adminguide_authentication_flow_link in document-attributes-product.adoc